src/plugins/blocks/utils.js

```diff
@@ -100,21 +100,44 @@ export function getBlockName(block, includeVariants = true) {
   return filteredClasses.length > 0 ? `${name} (${filteredClasses.join(', ')})` : name;
 }
 
-function getPreferedBackgroundColor() {
+export function getPreferedBackgroundColor(blockName) {
+  const defaultBackgroundColor = '#ff8012';
+  if (blockName === 'Section Metadata') {
+    return getComputedStyle(document.documentElement)
+      .getPropertyValue('--sk-section-metadata-table-background-color') || defaultBackgroundColor;
+  }
+
+  if (blockName === 'Metadata') {
+    return getComputedStyle(document.documentElement)
+      .getPropertyValue('--sk-metadata-table-background-color') || defaultBackgroundColor;
+  }
+
   return getComputedStyle(document.documentElement)
-    .getPropertyValue('--sk-table-bg-color') || '#ff8012';
+    .getPropertyValue('--sk-block-table-background-color') || defaultBackgroundColor;
 }
 
-function getPreferedForegroundColor() {
+export function getPreferedForegroundColor(blockName) {
+  const defaultBackgroundColor = '#ffffff';
+  if (blockName === 'Section Metadata') {
+    return getComputedStyle(document.documentElement)
+      .getPropertyValue('--sk-section-metadata-table-foreground-color') || defaultBackgroundColor;
+  }
+
+  if (blockName === 'Metadata') {
+    return getComputedStyle(document.documentElement)
+      .getPropertyValue('--sk-metadata-table-foreground-color') || defaultBackgroundColor;
```

Comment:
```suggestion
  const defaultForegroundColor = '#ffffff';
  if (blockName === 'Section Metadata') {
    return getComputedStyle(document.documentElement)
      .getPropertyValue('--sk-section-metadata-table-foreground-color') || defaultForegroundColor;
  }

  if (blockName === 'Metadata') {
    return getComputedStyle(document.documentElement)
      .getPropertyValue('--sk-metadata-table-foreground-color') || defaultForegroundColor;
```
Could be a bit confusing otherwise